Cubical/Algebra/CommRing/BinomialThm.agda

```diff
@@ -70,7 +70,7 @@ module BinomialThm (R' : CommRing ℓ) where
      ∑ xVec + ∑ yVec
   ≡⟨ cong (_+ ∑ yVec) (∑Last xVec) ⟩
      ∑ (xVec ∘ weakenFin) + xⁿ⁺¹ + (yⁿ⁺¹ + ∑ (yVec ∘ suc))
-  ≡⟨ solve3 _ _ _ _ ⟩
+  ≡⟨  solve3 _ _ _ _ ⟩
```

Comment:
so using `solve! R'` in place here is not possible? Is that because the ring solver can't parse `∑ (xVec ∘ weakenFin)` as a variable?

Reply:
This one is actually a reasonable problem: Inline use is not possible, because the generated term will have expressions involving stuff like weakenFin in it and agda won't be able to figure out what the implicit arguments of those terms are.